### ISSUE TITLE
Changing/Switching Igtimi accounts leads windStatus be empty

### ISSUE THREAD

**Issue by eclipse-sailing-analytics-bot** [eclipse-sailing-analytics/sailing-analytics#3719]
| Field | Value |
|---|---|
| **Reporter** | Steffen Wagner (`mail.sap@steffenwagner.com`) |
| **Assignee** | @axeluhl |
| **Product** | Sailing Race Analytics |
| **Component** | Server |
| **Version** | unspecified |
| **Hardware** | All |
| **OS** | All |
| **Severity** | normal |
| **Priority** | P2 |
| **CC (unmapped)** | Julian Mayland (`j.mayland@appweeve.com`) |
| **Created** | 2016-06-18T08:50:17Z |
| **Original Bug** | [Bug 3719](https://old-bugzilla.sapsailing.com/bugzilla/show_bug.cgi?id=3719) |

**Duplicate of:** #3758

---

If you change a igtimi account (e.g. same bots shared) on a running server (without restart), the windStatus servlet stays empty (altough with several restart trackings and reload wind receivers we get for a tracked race valid data from igtimi).

**Comment by eclipse-sailing-analytics-bot** [comment]
**Steffen Wagner (`mail.sap@steffenwagner.com`)** commented:

We were NOT able to reproduce on dev. Even two accounts with the same bot shared (and deleting randomly) is keeping the windStatus servlet filled. Don't know what happened on these two league servers....

**Comment by eclipse-sailing-analytics-bot** [comment]
*Steffen Wagner (`mail.sap@steffenwagner.com`)* changed status from **NEW** to **RESOLVED** (INVALID)

**Comment by eclipse-sailing-analytics-bot** [comment]
**@fmittag** commented:

We faced several issues this weekend around this topic while having 4 sailing leagues in parallel - all using different Igtimi accounts. He have seen some strange behavior in some league servers, but not all of them:
- Windstatus servlet did not show wind messages, but the trackedRaces received wind
- Removing and re-adding the Igtimi account worked on 1 server, but not at another one
- using the reloadWindReceiver parameter of the windStatusServlet to renew the Igtimi connections worked sometimes, sometimes not.

Analyzing the situation we found that there was a problem with one of the three Igtimi websocket connections. It seems the failover mechanism for broken connections causes some unexpected results.

**Comment by eclipse-sailing-analytics-bot** [comment]
**@fmittag** commented:

Created attachment 731
account_and_devices

![account_and_devices](https://github.com/eclipse-sailing-analytics/bugzilla-attachments/raw/main/3719/731_account_and_devices.jpg)
